### PR TITLE
Split commands out into separate files

### PR DESCRIPTION
**Pull request by joshnesbitt**

### CODE REVIEW

**Comment by joshnesbitt** [comment]
@mackstar can I get your opinion on this please? Thinking of splitting these into separate files for easier maintainability moving forward. Obviously still need to implement the functionality you previously added.

File: src/config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import constants from './bjconstants';
+import constants from './constants';
```
Comment (mackstar) [inline src/config.js]: I had this called constants before, it does cause problems on occasions, but I am happy to see if you hit that roadblock.

**Comment by mackstar** [comment]
Sure no worries. Am not precious about that stuff. Might be a case where this might seem potentially over-engineered. But is certainly simple and readable enough for me so I am happy with this.

File: src/config.js (hunk 1 above)
Comment (joshnesbitt) [inline src/config.js]: I thought there might have been a reason ;) I didn't hit this problem, but happy to rename.

**Comment by mackstar** [comment]
👍  Sure no worries.

**Comment by joshnesbitt** [comment]
Happy for me to merge this and open another PR for implementing the missing functionality? Might make the diff a bit more pleasant...

**Comment by mackstar** [comment]
No worries Josh - :shipit:

**Pull request merged**